src/app/adapters/fallbackAdapter.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import {
+  Graph,
+  NodeAddress,
+  type NodeAddressT,
+  EdgeAddress,
+} from "../../core/graph";
+import type {Repo} from "../../core/repo";
+
+import type {StaticPluginAdapter, DynamicPluginAdapter} from "./pluginAdapter";
+
+export const FALLBACK_NAME = "FALLBACK_ADAPTER";
+
+export class FallbackStaticAdapter implements StaticPluginAdapter {
```

Comment:
Implementation looks good. Do you think that any tests would be useful?

Reply:
Not really.